### ISSUE TITLE
Github Actions not running tests on Python 3.8 for Windows

### ISSUE THREAD

**Issue by blnicho** [Pyomo/pyomo#1257]
We need to figure out why 0 tests are run for Python 3.8 for Windows even though everything appears to run successfully.

**Comment by mrmundt** [comment]
@allevin - I'll look into this tomorrow.

**Comment by mrmundt** [comment]
I tested this on my local Windows VM with Miniconda (created a Python 3.8.1 enviroment). It turns out that it's a simple invocation change. With Python 3.8+ on Windows, nosetests requires the extra flag `--traverse-namespace` when there are multiple packages. It looks like other OS' can also use this command (I tried it on my local Mac on Python 2.7, 3.7, and 3.8) - it's just not required. 

I will make a new branch to test using Github Actions.
